lib/gds_api/publishing_api_v2.rb

```diff
@@ -1,6 +1,14 @@
 require_relative 'base'
 
 class GdsApi::PublishingApiV2 < GdsApi::Base
+  # Put a content item
+  #
+  # This creates a new draft item, which will be sent to the draft content store.
```

Comment:
Removed the extra explanation, as this is better described in the publishing-api docs.